README.md

```diff
@@ -43,9 +45,10 @@ PixiEditor started in 2018 and it's been actively developed since. We continuous
 
 Follow these instructions to get PixiEditor working on your machine.
 
-1. Download the .exe file from [here](https://github.com/flabbet/PixiEditor/releases)
-2. Open installer
-3. Follow installer instructions
+1. Download zipped file from our [official website](https://pixieditor.net/download/)
+2. Extract .zip archive
+3. Open installer
+4. Follow installer instructions
```

Comment:
Possibly better wording: 
1. Download the zipped installer from our [official website](https://pixieditor.net/download/)
2. Extract the installer from the archive
3. Launch it
4. Follow the steps in the installer to finish the installation

Reply:
Yeah, sounds better